dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/task/datax/DataxTask.java

```diff
@@ -462,7 +462,10 @@ public String loadJvmEnv(DataxParameters dataXParameters) {
 
         try {
             SQLStatementParser parser = DataxUtils.getSqlStatementParser(dbType, sql);
-            notNull(parser, String.format("database driver [%s] is not support", dbType.toString()));
+            if (parser == null) {
+                logger.warn("database driver [{}] is not support grammatical analysis sql", dbType);
+                return new String[0];
```

Comment:
Why not "return null;"

Reply:
I think there is no problem returning null, but the code smell check suggest we should use empty array to replace null.